dlt/helpers/dashboard/dlt_dashboard.py

```diff
@@ -617,16 +624,17 @@ def section_trace(
                     title_level=3,
                 )
             )
-            _result.append(
-                mo.accordion(
-                    {
-                        strings.trace_show_raw_trace_text: mo.ui.code_editor(
-                            json.dumps(dlt_trace, pretty=True),
-                            language="json",
-                        )
-                    }
-                )
-            )
+            # TODO: trace is not json serializable. I'd store it as such if that was easy
```

Comment:
I'm sending this to the platform though, I'll have to check what I do there to make it work, maybe as_dict or something.